zio-http/src/main/scala/zhttp/http/Scheme.scala

```diff
@@ -8,10 +8,12 @@ object Scheme       {
   def asString(self: Scheme): String = self match {
     case HTTP  => "http"
     case HTTPS => "https"
+    case WS    => "ws"
```

Comment:
Shouldn't we also have wss for secure websocket connection?

Reply:
I think we should do it in an incremental issue. What do you think?

Reply:
WSS is supported in the client. Let's have this done in this PR.

Reply:
done.